doc/api/http2.md

```diff
@@ -3028,8 +3028,11 @@ is finished.
 #### response.finished
 <!-- YAML
 added: v8.4.0
+deprecated: REPLACEME
 -->
 
+> Stability: 0 - Deprecated. Use [`response.writableEnded`][].
```

Comment:
This contradicts the above statement?
> Use [`response.writableFinished`][] or [`response.writablEnded`][] accordingly instead to avoid the ambigiuty.

Reply:
Not really, `writableFinished` is the correct property to use. The description should clarifies that some ambiguity might exist?

Reply:
So, if the user was using `.finshed` correctly, they can switch to `writableEnded`? But if they thought `.finished` meant something else rather than what it actually means, then `writableFinished` might be what they should use?

Reply:
@Trott: correct

Reply:
Pushed an update to try and further clarify.